pkg/extension/workspace/example/runtime/interfaces.go

```diff
@@ -16,7 +16,6 @@ package runtime
 
```

Comment:
the contents here are not "intefaces" anymore 😄

Reply:
haha renamed to types for now. hopefully a later PR we can delete even more of the intermediate types